docs/building.rst

```diff
@@ -481,6 +481,19 @@ If you enabled test in the configuration step, to run them, run::
 
 	ctest -j8
 
+building with VCPKG
+
+You can download and install libtorrent using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager:
```

Comment:
if you add another  colon ":" here, the block below will be a fixed with, preformatted, block.

Reply:
Updated, thanks!